index.js

```diff
@@ -62,11 +66,11 @@ module.exports = {
 			message: '`with` is disallowed in strict mode because it makes code impossible to predict and optimize.',
 			selector: 'WithStatement',
 		}],
-		'no-tabs': ['error', {
-			allowIndentationTabs: true,
-		}],
+		'no-tabs': 'off',
 		'no-underscore-dangle': ['off'],
 		'no-unused-vars': ['warn'],
+		'object-curly-newline': 'off',
+		'operator-linebreak': 'off',
```

Comment:
I prefer the `operator-linebreak: ["error", "before"]` [option](https://eslint.org/docs/rules/operator-linebreak#:~:text=Examples%20of%20correct%20code%20for%20this%20rule%20with%20the%20%22before%22%20option%3A)